docs/tensorboard_projector_plugin.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<p>"Tensorboard" -&gt; "TensorBoard"</p><p> </p><p>"from your tensorflow projects from the logs in the specified directory <code>log_dir</code> " is a bit confusing. Perhaps something like "from the logs in the specified<code>log_dir</code>  directory "?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3423/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>